src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -1261,18 +1261,20 @@ public void OnRigidityChanged(int desiredRigidity)
         if (previousRigidity == desiredRigidity)
             return;
 
-        int costPerStep = (int)Math.Min(Constants.MEMBRANE_RIGIDITY_COST_PER_STEP * CostMultiplier, 100);
+        var costPerStep = Math.Min(Constants.MEMBRANE_RIGIDITY_COST_PER_STEP * CostMultiplier, 100);
 
         var data = new RigidityActionData(desiredRigidity / Constants.MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO, Rigidity)
         {
             CostMultiplier = CostMultiplier,
         };
 
-        var cost = Editor.WhatWouldActionsCost(new[] { data });
+        // In some cases "theoreticalCost" might get rounded improperly
+        var theoreticalCost = Editor.WhatWouldActionsCost(new[] { data });
+        var cost = (int)Math.Ceiling(Math.Ceiling(theoreticalCost / costPerStep) * costPerStep);
 
-        if (cost > Editor.MutationPoints)
+        if (cost > Editor.MutationPoints && Editor.MutationPoints != 0)
```

Comment:
I think adding a comment here as well explaining that the mutation points being zero corner case is handled below.

BTW unrelated to this exact change, but what would you expect to happen once mutation points are converted to floats? That's a stalled PR and a pretty long-wanted fix for the code architecture.

Reply:
That's hard to tell but I guess the slider code would need some refactoring

Reply:
That's about what I expected... Hopefully this bugfix doesn't make that eventual work harder.